### ISSUE TITLE
Native VR Implementation does not work properly

### ISSUE THREAD

**Issue by ikriz** [cakeslice/Outline-Effect#29]
With Single-Pass Stereo Rendering and a Single Native Unity Camera.
Using the native Unity VR implementation the outline effect is broken.

**Comment by ikriz** [comment]
Tried rendering in Unity using the Split Stereo non-head mounted (VR mode) right eye still gets a weird offset)

**Comment by jaytee-sharp** [comment]
I can confirm that the outline effect is broken with Single-Pass Stereo Rendering. Tried with the HTC Vive and native VR implementation in Unity 5.5.2.

**Comment by cakeslice** [comment]
Yeah, sorry guys but this effect does not support VR in single pass mode. If someone knows how it can be implemented, please help!

**Comment by eobet** [comment]
Could there be any help gained by looking at the Silhouette shader supplied with Valve's SteamVR InteractionSystem? If you have the asset, it's located here:

SteamVR\InteractionSystem\Core\Shaders\Silhouette.shader

**Comment by xmardex** [comment]
This repo is Built-in image-effect era. For URP/HDRP a CommandBuffer on Camera.main does not inject into the pipeline the same way — you need a Renderer Feature (URP) or a Custom Pass (HDRP), and Unity 6 wants Render Graph.

I'm the author of Keyline, a from-scratch URP/HDRP hull outline (not a port of this repo). Demo: [https://unitycraft.org/4ssets/demo/keyline120/](https://unitycraft.org/4ssets/demo/keyline120/)
